test/cpp/end2end/cfstream_test.cc

```diff
@@ -393,21 +412,19 @@ TEST_P(CFStreamTest, ConcurrentRpc) {
   std::thread thd = std::thread([this, &rpcs_sent]() {
     void* got_tag;
     bool ok = false;
-    bool network_down = true;
     int total_completions = 0;
 
     while (CQNext(&got_tag, &ok)) {
       ++total_completions;
       GPR_ASSERT(ok);
       AsyncClientCall* call = static_cast<AsyncClientCall*>(got_tag);
+      int stream_id = GetStreamID(call->context);
       if (!call->status.ok()) {
-        gpr_log(GPR_DEBUG, "RPC failed: %s",
-                call->status.error_message().c_str());
+        gpr_log(GPR_DEBUG, "RPC with stream_id %d failed with error: %s",
+                stream_id, call->status.error_message().c_str());
         // Bring network up when RPCs start failing
-        if (network_down) {
-          NetworkUp();
```

Comment:
Could you explain why `NetworkUp()` is no longer needed?

Reply:
`NetworkUp()` is unnecessary because this test doesn't bring down the network. When adding this test case, I had incorrectly copy-pasted this code from the previous test case which does flap the network.

Reply:
OK That makes sence